app/_locales/es/messages.json

```diff
@@ -26,9 +34,18 @@
   "accountSelectionRequired": {
     "message": "¡Necesitas elegir una cuenta!"
   },
+  "active": {
+    "message": "Activo"
+  },
+  "activity": {
+    "message": "Actividad"
+  },
   "activityLog": {
     "message": "registro de actividades"
   },
+  "addAccount": {
+    "message": "Agregar una cuenta"
```

Comment:
Maybe I would use more "Añadir una cuenta". need confirmation by 1 more person to see if is worth changing

Reply:
I think "Agregar" makes sense here? At least if this comment is accurate: https://www.reddit.com/r/duolingo/comments/1alunr/what_is_the_difference_between_a%C3%B1adir_and_agregar/c8ylfwv/
The "Add" in this case is more the "add to a group/aggregate" case I think. Each account is independent.